src/parsing/test/testing.ts

```diff
@@ -48,6 +48,17 @@ interface TestCase {
             appendDot?: boolean;
         };
     };
+    expectedErrors?: {
+        text: string,
+        token: string,
+        line: number,
```

Comment:
We still have useless line field for some reason. Checkout this comment again, please!
https://github.com/gravity-ui/sql-autocomplete-parsers/pull/31/files